alws/alembic/versions/2fb5ffc7ff29_add_pgp_key_id.py

```diff
@@ -0,0 +1,29 @@
+"""add_pgp_key_id
+
+Revision ID: 2fb5ffc7ff29
+Revises: 76eec955d203
+Create Date: 2021-10-12 15:33:10.230841
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2fb5ffc7ff29'
+down_revision = '76eec955d203'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "builds",
+        sa.Column(
+            "pgp_key_id", sa.Text(), nullable=False
```

Comment:
Why it cannot be nullable? There will be some builds non-signed yet. Or defaults is an empty string?

Reply:
More than that, all our builds are not signed at the beginning